### ISSUE TITLE
A11y_Visual Studio Code_Welcome Screen_Info and Relationships:Voice Over is not announcing the label Names(Start, Recent, Getting Started)in Welcome page..

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-docs#4788]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#Infoand Relationships;#MAC;#AINotInScope;#VoiceOver;#Benchmark-VisualStudioCode-MAC-Aug2021

### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Info and Relationships

### Repro Steps:​
1.Open Visual Studio Code editor
2.Tab till Welcome tab and press enter
3.Verify whether the Voice Over is reading the label Names like(Start, Recent, Getting Started)in Welcome page or not.

### Actual Result:
Voice Over is not reading the label Names like(Start, Recent, Getting Started)in Welcome page.

### Expected Result: ​
Voiceover is should announce the Label Names like(Start, Recent, Getting Started)in Welcome page.

### User Impact:
Screen Reader users will find difficult in finding which elements are under which label if the Voiceover is not announcing the Label Names like like(Start, Recent, Getting Started)in Welcome page.

**Comment by Latha2564** [comment]
https://user-images.githubusercontent.com/89508906/131293988-025bd7f3-a9b8-4ae1-a68e-fcaf43ad2dfd.mov

**Comment by gregvanl** [comment]
@Latha2564 Issues related the VS Code product UI should be created in https://github.com/microsoft/vscode/issues.
This repository is for documentation or code.visualstudio.com website issues.
Moving to vscode repo.
